module_2/Facade/src/PersonFacade.php

```diff
@@ -0,0 +1,42 @@
+<?php
+class PersonFacade
+{
+    private $repository;
+
+    public function __construct(PersonRepository $repository)
+    {
+        $this->repository = $repository;
+    }
+
+    public function whoIsTheSmarter(string $person1Name, string $person2Name): Person
+    {
+        $person1 = $this->findPersonOrThrow($person1Name);
+        $person2 = $this->findPersonOrThrow($person2Name);
+
+        return $person1->getIq() >= $person2->getIq() ? $person1 : $person2;
```

Comment:
@Hlopec 
```suggestion
        return $person1->isSmarterThen($person2) ? $person1 : $person2;
```
That's who it should look like without Anemic Model applied.